addons/fastroping/XEH_postInit.sqf

```diff
@@ -8,6 +8,16 @@
     [FUNC(fastRopeServerPFH), 0, _this] call CBA_fnc_addPerFrameHandler;
 }] call CBA_fnc_addEventHandler;
 
+// Keybinds
+["ACE3 Vehicles", QGVAR(fastRope), localize LSTRING(Interaction_fastRope), {
+    if ([ACE_player, vehicle ACE_player] call FUNC(canFastRope)) then {
```

Comment:
I think we might want the following (probably don't need the in-vehicle check, but can't hurt)
```
if ((vehicle ACE_player) == player) exitWith {false};
if (!([ACE_player, vehicle ACE_player, []] call EFUNC(common,canInteractWith))) exitWith {false};
```

Reply:
Didn't you mean `(vehicle ACE_player) == ACE_player`?